src/main/java/org/apache/ibatis/migration/hook/scripts/HookScript.java

```diff
@@ -13,7 +13,7 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
-package org.apache.ibatis.migration.hook;
+package org.apache.ibatis.migration.hook.scripts;
```

Comment:
I can move these back to their original package if desired but there seems to be enough related code to warrant a new package